### ISSUE TITLE
Cannot install version 4 due to unmet requirements

### ISSUE THREAD

**Issue by malua** [dpa99c/cordova-plugin-inappbrowser-popup-bridge#8]
# Bug Report

### What is expected to happen?
Should install on cordova 9.0.1

### What does actually happen?
```
Unmet project requirements for latest version of cordova-plugin-inappbrowser-popup-bridge:
    cordova (9.0.1 in project, >100 required)
Fetching highest version of cordova-plugin-inappbrowser-popup-bridge that this project supports: 2.0.0 (latest is 4.0.1)

```

## Information
The problem is the cordova dependencies block in package.json

```
"engines": {
    "cordovaDependencies": {
      "0.2.3": {
        "cordova": ">=3.1.0"
      },
      "4.0.0": {
        "cordova": ">100"
      }
    }
  }
```

Thanks for this great plugin!

**Comment by dpa99c** [comment]
Oops, let's remove that silly dependency check - it's historical and no longer needed

**Comment by dpa99c** [comment]
Pushed out the fix in `@4.0.2` patch release
